cli/src/pcluster/resources/custom_resources/custom_resources_code/cleanup_resources.py

```diff
@@ -120,30 +120,12 @@ def _terminate_cluster_nodes(event):
             logger.info("Sleeping for 10 seconds to allow all instances to initiate shut-down")
             time.sleep(10)
```

Comment:
do we still need this?

Reply:
I think it's to avoid throttling in between for loop cycle of terminate calls

Reply:
no, I thought it was to cope with throttling, but that is already taken care by the for loop